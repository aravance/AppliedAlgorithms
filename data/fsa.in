477
-477
-0.000000
0.000000E-000
352341E
352341E+
-55.7E1
4
4.
-578
0.25
.25
.4356
+
-
*
