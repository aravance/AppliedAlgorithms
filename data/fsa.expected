S 2 2 2  accept
S 1 2 2 2  accept
S 1 2 3 4 4 4 4 4 4  accept
S 2 3 4 4 4 4 4 4 5 6 7 7 7  accept
S 2 2 2 2 2 2 5  reject
S 2 2 2 2 2 2 5 6  reject
S 1 2 2 3 4 5 7  accept
S 2  accept
S 2 3  reject
S 1 2 2 2  accept
S 2 3 4 4  accept
S  reject
S  reject
S 1  reject
S 1  reject
